bin/git-cml

```diff
@@ -77,20 +84,49 @@ def add(args):
     git_utils.add_file(args.file, repo)
 
 
-def init(args):
+def install(args):
     """
-    Set up git-cml repo by initializing .git/info/attributes and .git/config
+    Install git-lfs and initialize the git-cml filter driver
     """
-    repo = git_utils.get_git_repo()
-    config_writer = repo.config_writer()
+    git_utils.git_lfs_install()
+    config_writer = git.GitConfigParser(
+        git.config.get_config_path("global"), config_level="global", read_only=False
+    )
     config_writer.set_value('filter "cml"', "clean", "git-cml-filter clean %f")
     config_writer.set_value('filter "cml"', "smudge", "git-cml-filter smudge %f")
     config_writer.set_value('filter "cml"', "required", "true")
     config_writer.release()
 
-    with open(os.path.join(repo.working_dir, ".git/info/attributes"), "w+") as f:
-        f.write("*.pt filter=cml")
 
+def track(args):
+    """
+    Track a particular model checkpoint file with git-cml
+    """
+    repo = git_utils.get_git_repo()
+
+    # Initialize git lfs to track model files in .git_cml
+    model_dir = git_utils.create_git_cml_model_dir(repo, args.file)
+    git_utils.git_lfs_track(repo, model_dir)
+
+    gitattributes_file = git_utils.get_gitattributes_file(repo)
+    gitattributes = git_utils.read_gitattributes(gitattributes_file)
+
+    pattern_found = False
+    for line in gitattributes:
+        match = re.match("^\s*(?P<pattern>[^\s]+)\s+(?P<attributes>.*)$", line)
```

Comment:
Writing our own regex seems kind of error-prone - is there no other way to parse gitattributes files? E.g. some library that does it?